tests/test_bundle.py

```diff
@@ -0,0 +1,92 @@
+import shlex
+import time
+
+import pytest
+from helpers import get_ingress_ip_1dot4, get_ingress_url
+from lightkube.models.rbac_v1 import PolicyRule
+from lightkube.resources.core_v1 import Service
+from lightkube.resources.rbac_authorization_v1 import Role
+from pytest_operator.plugin import OpsTest
+
+USERNAME = "admin"
+PASSWORD = "foobar"
```

Comment:
In the past Rob raised that, even for something temporary, it is bad practice for us to embed usernames and passwords in our bundle tests.  I'm not as concerned but I see his point.  

The current bundle-kubeflow tests solve this problem by using [this](https://github.com/canonical/bundle-kubeflow/blob/920c7c8f191736cf2cb89f8cec2ff8f097a5e921/conftest.py) to intake credentials through either a command line argument or from an env variable.  Using command line alone is easier but can be awkward as it makes it easier to accidentally put the creds in CI logs.  You should be able to add that to your conftest.py without modifications

Reply:
Does it matter in testing though?

Reply:
Personally I don't think so, but in that other case we decided it did matter